src/state/swaps/swapsStore.ts

```diff
@@ -34,6 +36,74 @@ export interface SwapsState {
   setSource: (source: Source | 'auto') => void;
   degenMode: boolean;
   setDegenMode: (degenMode: boolean) => void;
+
+  // recent swaps
+  latestSwapAt: Map<ChainId, number>;
```

Comment:
is this prop used? do we need it?

Reply:
Not used but was in spec: https://www.notion.so/rainbowdotme/Show-Popular-in-Rainbow-Tokens-in-Swaps-on-App-BX-cc7b4a27f398427f8ede707c22943d94?pvs=4#eec96f7511ab4d3a9060a0980e14e787